dub.json

```diff
@@ -23,16 +23,16 @@
 	],
 	"buildTypes": {
 		"debug": {
-			"buildOptions": [ "unittests", "debugMode", "debugInfo" ]
+			"buildOptions": [ "unittests", "debugMode", "debugInfo", "coverage" ]
```

Comment:
Add to a new `configuration` instead, like `unittest-cov` and run tests with that.

Reply:
In the previous PR you said I should execute coverage with every build option, which I find totally unnecessary.

Reply:
> In the previous PR you said I should execute coverage with every build option, which I find totally unnecessary.

You shouldn't do `coverage` on `buildType` because every time you want to normally build your project, it will run with coverage enabled. Also, `coverage` on every `buildType` is a good thing because if you have some condition compilation scope that runs only with that flag, the code will be marked as covered. It is also a good thing to test with those flags as stuff can get broken when optimizations are applied.

Reply:
If I only execute `dub test` into doesn't. It just runs the `unittest` configuration and that's that.